packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts

```diff
@@ -60,9 +56,61 @@ export class BlockchainPropertiesService {
             registry.address
         );
 
-        await this.deploymentPropsRepo.save({ registry: registry.address });
+        await this.deploymentPropsRepo.save({ registry: registry.address, issuer: issuer.address });
+    }
+
+    async wrap(privateKey: string) {
+        const { registry, issuer } = await this.deploymentPropsRepo.get();
+
+        const { web3 } = await this.getProperties();
+
+        const signer: Signer = new Wallet(this.assure0x(privateKey), web3);
+        return {
+            web3,
+            registry: Contracts.factories.RegistryExtendedFactory.connect(registry, signer),
+            privateIssuer: undefined,
+            issuer: Contracts.factories.IssuerFactory.connect(issuer, signer),
+            activeUser: signer
+        };
     }
 
     private assure0x = (privateKey: string) =>
         privateKey.startsWith('0x') ? privateKey : `0x${privateKey}`;
 }
+
+@Injectable()
+export class BlockchainPropertiesService {
```

Comment:
is this Patryk's change or yours? afair Patryk validated if env are set in the constructor, you don't

Reply:
I changed it a little bit, the validation is still there